src/WordsDatabase.java

```diff
@@ -0,0 +1,37 @@
+import java.util.List;
+import java.util.Random;
+
+public class WordsDatabase {
+
+    private List<String> words;
```

Comment:
Two methods that should be in this class are - drawWord and addWord. Rest of them is useless